pybamm/models/submodels/interface/sei/ec_reaction_limited.py

```diff
@@ -0,0 +1,149 @@
+#
+# Class for reaction limited SEI growth
+#
+import pybamm
+from .base_sei import BaseModel
+
+
+class EcReactionLimited(BaseModel):
+    """Base class for reaction limited SEI growth.
+
+    Parameters
+    ----------
+    param : parameter class
+        The parameters to use for this submodel
+    domain : str
+        The domain of the model either 'Negative' or 'Positive'
+
+    **Extends:** :class:`pybamm.sei.BaseModel`
+    """
+
+    def __init__(self, param, domain):
+        super().__init__(param, domain)
+
+    def get_fundamental_variables(self):
+
+        L_sei = pybamm.Variable(
+            "Total " + self.domain.lower() +
+            " electrode sei thickness",
+            domain=self.domain.lower() + " electrode",
+            auxiliary_domains={"secondary": "current collector"},
+        )
+        j_sei = pybamm.Variable(
+            self.domain + " electrode sei interfacial current density",
+            domain=self.domain.lower() + " electrode",
+            auxiliary_domains={"secondary": "current collector"},
+        )
+        c_ec = pybamm.Variable(
+            self.domain + " electrode EC surface concentration",
+            domain=self.domain.lower() + " electrode",
+            auxiliary_domains={"secondary": "current collector"},
+        )
+        L_sei_av = pybamm.x_average(L_sei)
+        j_sei_av = pybamm.x_average(j_sei)
+        c_ec_av = pybamm.x_average(c_ec)
+
+        L_scale = pybamm.sei_parameters.L_sei_0_dim
+        # in this model the scale is identical to the intercalation current
+        j_scale = pybamm.sei_parameters.j_scale
```

Comment:
`j_scale` is different in the negaive and positive electrodes so you need to be a bit careful here (though at the moment an error will be raised if the positive electrode sei model isn't `NoSEI`, so it might be ok)

Reply:
Modified in the file to explicitly recover the negative scale.